packages/replay-next/pages/variables.css

```diff
@@ -137,16 +135,17 @@
   --theme-base-60: #c3c6c8;
 
   /* Tabs (Dark) */
-  /* Migration note: removed from src/devtools/client/themes/variables.css */
   --tab-bgcolor: var(--theme-base-100);
   --tab-color: var(--theme-base-40);
   --tab-hover-bgcolor: var(--theme-base-85);
   --tab-hover-color: var(--tab-selected-color);
   --tab-selected-bgcolor: var(--theme-base-95);
   --tab-selected-color: #fff;
+  /* --tab-standard-color: ; to add */
+  /* --tab-standard-hover-bgcolor: ; to add */
+  /* --tab-bgcolor: ; */
```

Comment:
Curious why we aren't at least adding some placeholder colors for these dark theme values?

Reply:
Didn't  want to accidentally introduce a new color  that causes a regression. Will be fixing these colors in another pr.